woocommerce/woocommerce-android
# Issue/9796 filter product status

Closes: #9796 


### Description

This PR fixes a bug in the product listing screen where the "Product Status" filter was not visible.

#### Reason
We were passing `NonPublishedProducts` product filter to the product filter screen. Applying that restriction basically says that we should only display published products on the product listing screen. When there's a restriction like this in the code for the product listing screen, it doesn't make sense to display a `product status` filter.

##### Fix
Remove the `NonPublishedProducts` product filter restriction on the product filtering screen.

### Testing instructions


Open the product listing screen, tap on the filter button on the top, and you must see the `product status` filter option.

### Images/gif
